hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java

```diff
@@ -90,10 +90,13 @@ public static Instant getTestStartTime() {
   }
 
   /**
+   * @deprecated use {@link org.junit.jupiter.api.io.TempDir} instead.
+   *
    * Get the (created) base directory for tests.
    *
    * @return the absolute directory
    */
```

Comment:
Shouldn't `@deprecated` javadoc tag be grouped with the other tags, at the end?

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#orderoftags